projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-block-editor-assets.php

```diff
@@ -469,6 +472,149 @@ private function setup_block_editor_screen() {
 		}
 	}
 
+	/**
+	 * Removes hooks from problematic plugins that cause errors in this endpoint.
+	 *
+	 * Some plugins conditionally load admin-only code based on is_admin(), which
+	 * returns false in REST API contexts. When these plugins hook into
+	 * enqueue_block_editor_assets without checking the context, they may call
+	 * undefined functions that were never loaded, causing fatal errors.
+	 *
+	 * This method preemptively removes hooks from known problematic plugins before
+	 * the enqueue_block_editor_assets action fires, preventing fatal errors.
+	 */
+	private function remove_problematic_plugin_hooks() {
+		global $wp_filter;
+
+		// List of problematic plugins and their callback patterns
+		$problematic_plugins = array(
+			'wpforms-lite' => array(
+				'enqueue_block_editor_assets',
+			),
+		);
+
+		foreach ( $problematic_plugins as $plugin_slug => $hook_names ) {
+			foreach ( $hook_names as $hook_name ) {
+				if ( ! isset( $wp_filter[ $hook_name ] ) ) {
+					continue;
+				}
+
+				// Iterate through all priorities for this hook
+				foreach ( $wp_filter[ $hook_name ]->callbacks as $priority => $callbacks ) {
+					foreach ( $callbacks as $callback_data ) {
+						// Check if this callback belongs to the problematic plugin
+						if ( $this->is_callback_from_plugin( $callback_data['function'], $plugin_slug ) ) {
+							remove_action( $hook_name, $callback_data['function'], $priority );
+						}
+					}
+				}
+			}
+		}
+	}
+
+	/**
+	 * Checks if a callback function belongs to a specific plugin.
+	 *
+	 * @param callable|array $callback The callback to check.
+	 * @param string         $plugin_slug The plugin slug to match against.
+	 * @return bool True if the callback belongs to the plugin, false otherwise.
+	 */
+	private function is_callback_from_plugin( $callback, $plugin_slug ) {
+		// Handle array-based callbacks: object methods or static class methods
+		if ( is_array( $callback ) && count( $callback ) === 2 ) {
+			$class_or_object = $callback[0];
+
+			// Handle object method callbacks [object, 'method_name']
+			if ( is_object( $class_or_object ) ) {
+				try {
+					$reflection = new ReflectionClass( $class_or_object );
+					$file_path  = $reflection->getFileName();
+					if ( $file_path && $this->is_file_from_plugin( $file_path, $plugin_slug ) ) {
+						return true;
+					}
+				} catch ( ReflectionException $e ) { // phpcs:ignore Generic.CodeAnalysis.EmptyStatement.DetectedCatch
+					// Failed to reflect on this object (e.g., internal class, anonymous class).
+					// Continue checking other callback types.
+				}
+			}
+
+			// Handle static method array callbacks ['ClassName', 'method_name']
```

Comment:
This also could be removed since we could just directly call 
something like 
```
remove_action( 'hook', ['ClassName', 'method_name'] )
```

Reply:
Good point. Addressed this in 0284583885324e73cfab793dede58d46cb4b24ec.